src/main/java/com/weirddev/testme/intellij/ui/customizedialog/CustomizeTestDialog.java

```diff
@@ -0,0 +1,184 @@
+package com.weirddev.testme.intellij.ui.customizedialog;
+
+import java.util.*;
+import java.util.List;
+
+import javax.swing.*;
+
+import com.intellij.psi.*;
+import com.intellij.psi.util.InheritanceUtil;
+import com.intellij.psi.util.PsiUtil;
+import com.weirddev.testme.intellij.builder.MethodFactory;
+import com.weirddev.testme.intellij.common.utils.PsiMethodUtils;
+import com.weirddev.testme.intellij.generator.MockBuilderFactory;
+import com.weirddev.testme.intellij.template.FileTemplateContext;
+import com.weirddev.testme.intellij.template.TemplateRegistry;
+import com.weirddev.testme.intellij.template.context.*;
+import com.weirddev.testme.intellij.utils.ClassNameUtils;
+import com.weirddev.testme.intellij.utils.JavaPsiTreeUtils;
+import com.weirddev.testme.intellij.utils.JavaTypeUtils;
+import org.jetbrains.annotations.NotNull;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.DialogWrapper;
+import com.intellij.openapi.util.NlsContexts;
+import com.intellij.refactoring.ui.MemberSelectionTable;
+import com.intellij.refactoring.util.classMembers.MemberInfo;
+import com.intellij.ui.ScrollPaneFactory;
+
+/**
+ *
+ * dialog for user to check fields mockable and methods testable
+ *
+ * @author huangliang
+ */
+public class CustomizeTestDialog extends DialogWrapper {
+
+    private final PsiClass myTargetClass;
+    private final MemberSelectionTable myMethodsTable = new MemberSelectionTable(Collections.emptyList(), null);
+    private final MemberSelectionTable myFieldsTable = new MemberSelectionTable(Collections.emptyList(), null);
+    private final FileTemplateContext fileTemplateContext;
+
+    public CustomizeTestDialog(@NotNull Project project, @NotNull @NlsContexts.DialogTitle String title,
+        PsiClass targetClass, FileTemplateContext fileTemplateContext) {
+        super(project, true);
+        myTargetClass = targetClass;
+        this.fileTemplateContext = fileTemplateContext;
+        initMethodsTable(myTargetClass);
+        initFieldsTable(myTargetClass, fileTemplateContext.getFileTemplateDescriptor().getDisplayName());
```

Comment:
please replace with 
` myFieldsTable = new MemberSelectionTable( initFieldsTable(myTargetClass, fileTemplateContext.getFileTemplateDescriptor().getDisplayName()), null);`
while following line can be removed from _initFieldsTable()_ -  `myFieldsTable.setMemberInfos(result);` , along with the inline initialization statement